### PR TITLE
Updates webhook concept piece

### PR DESCRIPTION
**Pull request by remotekieran**
I updated the content for the webhooks piece - please just review for any errors.

I've updated the necessary things (i.e. the webhooks card on the concepts page, helper) on the website repo too - that will show once I push my pricing copy changes.

### CODE REVIEW

File: content/concepts/webhooks.textile [hunk 1]
```diff
@@ -37,53 +44,68 @@ __WebHooks are the HTTP APIs built by the vendors and registered at the company
...
+A number of high-profile service providers deliver key capabilities through the use of webhooks, which you can leverage. Let’s look at a couple of examples.
 
 h2(#webhooks-examples). Webhooks - real-world examples
 
 h3. 1. Marketing platforms
 
-Mailchimp is one of many platforms that relies on webhooks for everyday events such as subscribing, unsubscribing and profile changes. This is useful if you’re using MailChimp to signup users from your website to your newsletter but you also wanted to pass that data to your CRM system. SendGrid also offers webhooks for informing users when transactional emails are delivered, opened or unintentionally not delivered to the recipient.
+MailChimp is one of many platforms that relies on webhooks to support everyday tasks, such as subscribing to and unsubscribing from to email lists, as well as user profile changes. This is useful if you’re using MailChimp as the mechanism to sign website users up to your newsletter, but you also want to pass the data to your CRM system.
```
Comment (tomczoink) [inline content/concepts/webhooks.textile]: `unsubscribing from to email lists,` => `unsubscribing from email lists` I think?

File: content/concepts/webhooks.textile [hunk 2]
```diff
@@ -37,53 +44,68 @@ __WebHooks are the HTTP APIs built by the vendors and registered at the company
...
 h3. 2. Payments
 
-Stripe offers several events that can be used as webhooks within your application. Webhooks provide notifications of successful or failed charges.
+Stripe offers "several events that can be used as webhooks":https://stripe.com/docs/webhooks within your application. Webhooks provide notifications of successful or failed charges.
 
-h3. 3. Platforms
+h3. 3. Collaboration platforms
 
-An everyday example of a platform using webhooks to enable communication between users is Github. Github uses webhooks for events related to the repository. Actions like code push, repository created/deleted and others all rely on webhooks which also configure the CI/CD process according to it.
+GitHub is a great example of a collaborationplatform using webhooks to facilitate communication between users. Github uses webhooks for events related to the repository. Actions including code push, repository created and repository deleted all rely on webhooks, which also configure the associated continuous integration/continuous delivery (CI/CD) processes.
```
Comment (tomczoink) [inline content/concepts/webhooks.textile]: `collaborationplatform` => `collaborationplatform` or is that a thing?

File: content/concepts/webhooks.textile [hunk 3]
```diff
@@ -37,53 +44,68 @@ __WebHooks are the HTTP APIs built by the vendors and registered at the company
...
 
 h3. 4. CRM Platforms
 
-Lead generation platform HubSpot provides webhooks through the Ably platform for notifying users, in real time, about specific events their customers have opted to hear about. This could be company creation, deal creation, contact information changes, replies to messages, and so on. Realtime webhooks infrastructure means users can listen to selected notifications and store these in their system or dashboard to keep system synchronized with HubSpot’s account changes. Here, realtime is an important factor as it ensures a delicate interplay of microservices remain in sync.
+Lead-generation platform HubSpot provides webhooks through the Ably platform for notifying users, in real time, about specific events their customers have opted to hear about. This could be company-creation, deal-creation, contact information changes, replies to messages, and so on. Realtime webhooks infrastructure means users can listen to selected notifications and store these in their system or dashboard, to keep their system synchronised with HubSpot’s account changes. Here, realtime is an important factor, because it ensures a delicate interplay of microservices remains in sync.
```
Comment (tomczoink) [inline content/concepts/webhooks.textile]: `synchronised` => `synchronized` for that 'merica feel.

**Comment by remotekieran** [comment]
Changed, thanks @tomczoink

**Pull request merged**